com.hazelcast.spring.cache.TestCacheManager.testBean_withTTL failure [HZ-996]

on master

Failed on [Hazelcast-pr-builder/7140](http://jenkins.hazelcast.com/job/Hazelcast-pr-builder/7140/testReport/junit/com.hazelcast.spring.cache/TestCacheManager/testBean_withTTL/)

Stacktrace:
```
java.lang.AssertionError: expected:<ali> but was:<null>
	at com.hazelcast.spring.cache.TestCacheManager.testBean_withTTL(TestCacheManager.java:87)
```

Standard output:
```
BuildInfo right after testBean_withTTL(com.hazelcast.spring.cache.TestCacheManager): BuildInfo{version='5.0-SNAPSHOT', build='20210914', buildNumber=20210914, revision=d8a103b, enterprise=false, serializationVersion=1}
Hiccups measured while running test 'testBean_withTTL(com.hazelcast.spring.cache.TestCacheManager):'
10:45:05, accumulated pauses: 2052 ms, max pause: 78 ms, pauses over 1000 ms: 0


No metrics recorded during the test
```

_5.0.z_ (commit e70f143872aa5beb83d53111e17fd1366de15b27)

Failed on PR builder: http://jenkins.hazelcast.com/job/Hazelcast-pr-builder/7798/testReport/junit/com.hazelcast.spring.cache/TestCacheManager/testBean_withTTL/

<details><summary>Stacktrace:</summary>

```
java.lang.AssertionError: expected:<ali> but was:<null>
	at com.hazelcast.spring.cache.TestCacheManager.testBean_withTTL(TestCacheManager.java:87)
```

</details>

<details><summary>Standard output:</summary>

```
BuildInfo right after testBean_withTTL(com.hazelcast.spring.cache.TestCacheManager): BuildInfo{version='5.0.1-SNAPSHOT', build='20211012', buildNumber=20211012, revision=554cc59, enterprise=false, serializationVersion=1}
Hiccups measured while running test 'testBean_withTTL(com.hazelcast.spring.cache.TestCacheManager):'
21:41:55, accumulated pauses: 125 ms, max pause: 28 ms, pauses over 1000 ms: 0


No metrics recorded during the test
```

</details>

--- a/hazelcast-spring-tests/src/test/java/com/hazelcast/spring/cache/TestCacheManager.java
+++ b/hazelcast-spring-tests/src/test/java/com/hazelcast/spring/cache/TestCacheManager.java
@@ -124,6 +124,8 @@ public class TestCacheManager extends HazelcastTestSupport {
         Collection<String> test = cacheManager.getCacheNames();
         assertContains(test, testMap);
         testInstance.shutdown();
+        // Wait for the cluster to scale down, so it doesn't affect other tests
+        assertClusterSizeEventually(1, instance);
     }
 
     public static class DummyBean implements IDummyBean {

--- a/hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/cache/cacheManager-applicationContext-hazelcast.xml
+++ b/hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/cache/cacheManager-applicationContext-hazelcast.xml
@@ -44,7 +44,7 @@
                               class-name="com.hazelcast.spring.cache.CacheMapLoader"/>
             </hz:map>
             <hz:map name="null-map"/>
-            <hz:map name="map-with-ttl" time-to-live-seconds="1">
+            <hz:map name="map-with-ttl" time-to-live-seconds="2">
             </hz:map>
         </hz:config>
     </hz:hazelcast>

